### PR TITLE
Remote read blog post: fixed 2 typos.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartek Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: content/blog/2019-10-10-remote-read-meets-streaming.md [hunk 1]
```diff
@@ -326,8 +326,7 @@ there are still few items to do in order to fully get advantage from the extende
 
 To sum up, the main benefits of chunked, streaming of remote read are:
 
-* Both client and server are capable to use **practically constant memory size and per request**. This is because Prometheus processes
-and sends just single small frames one by one instead of the whole response during remote read. This massively helps with
+* Both client and server are capable to use **practically constant memory size and per request**. This is because Prometheus process sends just single small frames one by one instead of the whole response during remote read. This massively helps with
```
Comment (brian-brazil) [inline content/blog/2019-10-10-remote-read-meets-streaming.md]: are capable of using

the Prometheus sends

Comment (beorn7) [reply content/blog/2019-10-10-remote-read-meets-streaming.md]: @bwplotka gentle ping. I agree with Brian's suggestions here. If you could apply them to this PR, we could merge it.

(In different news: Shouldn't we better remove the "and" in "constant memory size and per request"?)

Comment (brian-brazil) [reply content/blog/2019-10-10-remote-read-meets-streaming.md]: Yes, that and doesn't belong

**Comment by bwplotka** [comment]
Done, thanks for review :+1:

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**